### ISSUE TITLE
Keyword varying carried over in precision qualifier example

### ISSUE THREAD

**Issue by ndhb** [danginsburg/opengles3-book#19]
In printed version p. 120:

varying lowp vec4 color;

Could replace it with keyword out or remove it.

**Comment by danginsburg** [comment]
Yes, this was a mistake, thanks for catching it.  As you note, varying has been deprecated.
